eeps/eep-0010.md

```diff
@@ -314,12 +311,12 @@ Same as characters_to_binary(ML,InEncoding,unicode).
 
 Types:
 
-- ML := A mixed list of integers or binaries corresponding to the
```

Comment:
This, and surely more of the same, below, is warned about when you start mixing styles.